### ISSUE TITLE
Playback of entering 2nd voice is too loud

### ISSUE THREAD

**Issue by Grillzombie** [musescore/MuseScore#25221]
### Issue type

General playback bug

### Description with steps to reproduce

1. Set up score with wizard Tenor voice and Bass voice
2. Enter some notes in first voice of Tenor
3. Enter some notes in second voice of Tenor -> these are played much louder

### Supporting files, videos and screenshots

https://github.com/user-attachments/assets/a87f1652-ce4d-4087-b478-c935702ba8df

### What is the latest version of MuseScore Studio where this issue is present?

MuseScore Studio version (64-bit): 4.4.3-242921234, revision: github-musescore-musescore-c84617c

### Regression

I was unable to check

### Operating system

OS: Linux Mint 22, Arch.: x86_64, 

### Checklist

- [X] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [X] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [X] I have attached all requested files and information to this report
- [X] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by MarcSabatella** [comment]
I can't reproduce this, but it does seem suspiciously similar to https://github.com/musescore/MuseScore/issues/25087 which I *can* reproduce, but only in that one score, and only temporarily (problem goes away on save/reload).

**Comment by bkunda** [comment]
@Grillzombie can you please let us know what specific sound you are using for the tenors in the mixer? 

Does it reproduce when using MS Basic or VST as well?

**Comment by Grillzombie** [comment]
Hello @bkunda, i'm using the MS Basic sound in this example, build from scratch

**Comment by zacjansheski** [comment]
Regression occured between 4.3.0 and 4.3.1

Seems like Voice 1 volume was reduced in volume but not the other voices


https://github.com/user-attachments/assets/62fad1f9-379e-42ea-bb61-8850f98efa9a

**Comment by MarcSabatella** [comment]
Yes, I see this now.  I assume then it is connected to https://github.com/musescore/MuseScore/pull/22844, but it's not really obvious why that would have affected only voice 1, so I'm assuming the issue is actually elsewhere but simply wasn't noticeable before this change.
